tests/test_adjoint.py

```diff
@@ -28,17 +28,16 @@ def setup_method(self, method):
     @pytest.mark.parametrize('mkey, shape, kernel, space_order, nbpml', [
         # 1 tests with varying time and space orders
         ('layers', (60, ), 'OT2', 4, 10), ('layers', (60, ), 'OT2', 8, 10),
-        ('layers', (60, ), 'OT4', 4, 10), ('layers', (60, ), 'OT4', 8, 10),
+        ('layers', (60, ), 'OT4', 4, 10),
         # 2D tests with varying time and space orders
         ('layers', (60, 70), 'OT2', 4, 10), ('layers', (60, 70), 'OT2', 8, 10),
-        ('layers', (60, 70), 'OT2', 12, 10), ('layers', (60, 70), 'OT4', 4, 10),
-        ('layers', (60, 70), 'OT4', 8, 10), ('layers', (60, 70), 'OT4', 12, 10),
+        ('layers', (60, 70), 'OT2', 12, 10), ('layers', (60, 70), 'OT4', 8, 10),
         # 3D tests with varying time and space orders
         ('layers', (60, 70, 80), 'OT2', 4, 10), ('layers', (60, 70, 80), 'OT2', 8, 10),
         ('layers', (60, 70, 80), 'OT2', 12, 10), ('layers', (60, 70, 80), 'OT4', 4, 10),
-        ('layers', (60, 70, 80), 'OT4', 8, 10), ('layers', (60, 70, 80), 'OT4', 12, 10),
         # Constant model in 2D and 3D
         ('constant', (60, 70), 'OT2', 8, 14), ('constant', (60, 70, 80), 'OT2', 8, 14),
+        ('constant', (60, 70), 'OT4', 12, 14), ('constant', (60, 70, 80), 'OT4', 4, 14),
```

Comment:
Note for reviewers:

So yes I removed some but if you look at it I changed the space orders too so that it's all still tested but should remove quite some time from CI as OT4 is like 90% of this test